index.js

```diff
@@ -81,6 +82,7 @@ function startSauceConnect (done) {
 			}
```

Comment:
Actually, I think execution will fall through here and hit the following code so you need to do else.